tests/test-cluster.py

```diff
@@ -558,28 +573,63 @@ def test_no_cert_reissue_in_nodes(self):
             lock_files = vm.run("ls /var/snap/microk8s/current/var/lock/")
             assert "no-cert-reissue" in lock_files.decode()
 
+    @pytest.mark.skipif(
+        # If the host system does not have IPv6 support or is not configured for IPv6, it won't be able to create VMs with IPv6 connectivity.
+        backend != "lxc" or not is_ipv6_configured,
```

Comment:
Multipass also supports IPv6, why are we explicit to lxc here?

Reply:
Due to the use of commands like these (https://github.com/canonical/microk8s/pull/4079/files#diff-8266c515875225e782094da6227c5ca720828f786e9f4c793b12e025e88f2e7aR585) which assume a lxc container for tests.

Reply:
Then we should implement this push in the VM class and we make the test below backend agnostic (as with the rest of the test) .